### DISCUSSION TITLE
2009/09/04/%E4%B8%8B%E4%B8%80%E4%B8%AA%E6%9C%9F%E5%BE%85%E7%AA%81%E7%A0%B4%E7%9A%84%E4%BA%A7%E4%B8%9A

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#274] [Announcements]
# 2009/09/04/%E4%B8%8B%E4%B8%80%E4%B8%AA%E6%9C%9F%E5%BE%85%E7%AA%81%E7%A0%B4%E7%9A%84%E4%BA%A7%E4%B8%9A

这几天我躺在床上的时候就在想，啥行业最有潜力呢，下一个吸引最多目光和投资的领域是啥呢？

https://blog.qizhen.xyz/2009/09/04/%E4%B8%8B%E4%B8%80%E4%B8%AA%E6%9C%9F%E5%BE%85%E7%AA%81%E7%A0%B4%E7%9A%84%E4%BA%A7%E4%B8%9A

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Smart
2009年10月13日 08:22 编辑
一定不是信息产业，

回复
Qizhen
2009年09月12日 13:54 编辑
垂直轴风力发电机不是接收垂直方向风的，而是说风水平吹来的时候，它沿着垂直方向旋转。这就意味着任何方向的风对它来说都一样，最有效的克服了风在方向上的不稳定性。现在垂直轴风力发电机的效率和水平轴发电机差不多，只是设计起来跟困难。风力和太阳能都不稳定，都需要配合蓄电池一起使用的。

回复
竹贤
2009年09月12日 13:04 编辑
风能最大的问题是不稳定。另外，垂直方向的风有多大？

回复
佳
2009年09月8日 15:33 编辑
你打字速度肯定特别快，能把想到的那么多东西都记录下来~~太佩服了。我怎么都忘记自己什么时候加你为好友的呢。。

回复
建权
2009年09月6日 15:22 编辑
不光技术好，而且文笔也很赞：）

回复
